apps/ssr-tests-v9/package.json

```diff
@@ -16,6 +16,9 @@
     "test": "jest --passWithNoTests",
     "type-check": "tsc -b tsconfig.json"
   },
+  "dependencies": {
+    "@fluentui/react-components": "^9.0.0-rc.14"
```

Comment:
Is not really needed. Is added to be triggered as a part of `react-components` build.